examples/streaming/stockcrawler/README.md

```diff
@@ -0,0 +1,19 @@
+How to use
+===================
+1. Start local cluster, 
+  ```
+  bin/local -ip 127.0.0.1 -port 3000
+  ```
+2. Submit the stock crawler
+  ```
+  bin\gear app -jar examples\gearpump-examples-assembly-0.3.2-SNAPSHOT.jar org.apache.gearpump.streaming.examples.stock.main.Stock -master 127.0.0.1:3000
```

Comment:
Do you think including a specified version number will not work for future version? Usually I'd use `examples\gearpump-examples-assembly-*.jar`